### PR TITLE
Upgrade to work with Rails 8

### PR DESCRIPTION
**Pull request by robzolkos**
Rails 8 was released today.

### CODE REVIEW

**Comment by Fivell** [comment]
@robzolkos  thanks can you please also add configuration for running rails 8 pipeline for tests here  https://github.com/didww/credit_card_validations/blob/master/.github/workflows/build.yml  ?

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -21,11 +21,16 @@ jobs:
...
         - ruby-version: 2.7
-          rails: '7.2.0'
+          rails:
+            - '7.2.0'
+            - '8.0.0'
         - ruby-version: '3.0'
-          rails: '7.2.0'
+          rails:
+            - '7.2.0'
+            - '8.0.0'
```
Comment (y-yagi) [inline .github/workflows/build.yml:33]: Rails 8.0.0 only supports Ruby >= 3.2.0.
https://rubygems.org/gems/rails/versions/8.0.0

So we need to add the Rails 8.0 and Ruby 3.1 matrix to the `exclude` list.

**Comment by Fivell** [comment]
will try to fix in master

**Comment by Fivell** [comment]
@robzolkos  released , thanks

**Pull request merged**